docs/data/joy/customization/introduction/introduction.md

```diff
@@ -0,0 +1,118 @@
+# Introduction
+
+<p class="description">Learn the ways to customize Joy UI.</p>
+
+<!-- The purpose of this page is to give the overall customization alternatives to developers without providing too much details -->
+<!-- Some examples and demos are provided to give the sense of what it looks like and then lead each part to another page for more technical details and more examples -->
+
+The customization can be categorized in 2 buckets:
+
+- Global customization (theming)
+- One-off customization (per instance)
+
+## Theming
+
+If you want to apply the style to every instance of the components, theming is the right choice. The benefit of theming is that your implementation does not change when you switch between themes.
+
+Here are some examples that replicate the popular designs (only the light mode):
+
+{{"demo": "ButtonThemes.js", "hideToolbar": true}}
+
+Theming can be categorized into 2 parts, **tokens** and **components**.
+
+### Tokens theming
+
+The tokens (design tokens) are meaningful variables that refer to the generic usage of the application. The idea of the tokens is to create consistency for the application.
+
+:::warning
```

Comment:
nit: the blue underline on the link looks weird:

![image](https://user-images.githubusercontent.com/4512430/170458252-ca69f515-bd40-4c8d-ae05-de5bf4bd3f29.png)

Reply:
we're all designers!